docker/Dockerfile.sidecar.alpine

```diff
@@ -132,6 +36,9 @@ RUN set -ex \
     && mkdir /lib64 \
     && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
 
+RUN apk --no-cache add openssl
+RUN wget -O /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64
+RUN chmod +x /usr/bin/dumb-init
 
```

Comment:
it apparently does not because the alpine image fails to build without adding openssl